.claude/skills/git-rebase-sync/scripts/smoke_test.py

```
@@ -0,0 +1,199 @@
+    found_any = False
+    for header in SECTION_HEADERS:
+        start = text.find("\n" + header + "\n")
+        if start == -1:
+            continue
+        found_any = True
+        end = text.find("\n## ", start + 1)
+        section = text[start : end if end != -1 else len(text)]
+
+        sub_starts = [m.start() for m in re.finditer(r"^### ", section, re.MULTILINE)]
+        if sub_starts:
+            bounds = sub_starts + [len(section)]
```

Ruff (0.16.1)

[warning] Consider `[*sub_starts, len(section)]` instead of concatenation

Replace with `[*sub_starts, len(section)]`

(RUF005)